### PR TITLE
update versioning in setup, requirements and Dockerfile

### PR DESCRIPTION
**Pull request by charliedrage**
relates to issue #226

### CODE REVIEW

**Comment by charliedrage** [comment]
with this change, we can also remove 
```bash
pip install -r ./requirements.txt && \ 
``` 
from the list of commands when building / tagging images.

File: setup.py [hunk 1]
```diff
@@ -40,16 +40,16 @@ def _install_requirements():
     return requirements
 
 setup(
-    name='atomicapp',
-    version='0.1.9',
```
Comment (goern) [inline setup.py]: could you read the value from https://github.com/projectatomic/atomicapp/blob/master/atomicapp/constants.py#L26 ?

Comment (charliedrage) [inline setup.py]: wouldn't it be easier to have a file in the root folder named VERSION that you can read it from? seems that whenever a new version comes out you have to s/0.1.9/0.2.0/ every file

Comment (charliedrage) [inline setup.py]: also, it would make it easier for contributors which version they're working on by simply
```bash
cat VERSION
```
the dir

**Comment by vpavlin** [comment]
Hmm..we should also fix a project url and maybe change the author to something more generic than my name:)

File: Dockerfile [hunk 2]
```diff
@@ -11,18 +11,16 @@ WORKDIR /opt/atomicapp
...
 
 # lets install pip, and gcc for the native extensions
 # and remove all after use
-RUN yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc && \
-    python setup.py install && \
-    pip install -r ./requirements.txt && \
-    yum remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
+RUN yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc
+RUN python setup.py install
+RUN yum remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
```
Comment (charliedrage) [inline Dockerfile]: Reason for multiple lines of RUN is to speed up cache when doing 'docker build -t projectatomic .'

**Comment by charliedrage** [comment]
will need an ack from @goern before merging, as there are some Dockerfile changes :+1:

File: Dockerfile [hunk 3]
```diff
@@ -11,18 +11,16 @@ WORKDIR /opt/atomicapp
...
 # add EPEL repo for pip
 RUN echo -e "[epel]\nname=epel\nenabled=1\nbaseurl=https://dl.fedoraproject.org/pub/epel/7/x86_64/\ngpgcheck=0" > /etc/yum.repos.d/epel.repo
 
 # lets install pip, and gcc for the native extensions
 # and remove all after use
-RUN yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc && \
-    python setup.py install && \
-    pip install -r ./requirements.txt && \
-    yum remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
+RUN yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc
```
Comment (vpavlin) [inline Dockerfile]: You need to add `&& yum clean all` at the end, otherwise we will end up carrying worthless data in the image. (i.e. every RUN creates a layer and every layer where yum is called needs to clear the cache otherwise it stays in layer although it's removed the next layer)

File: Dockerfile.fedora [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
 # add EPEL repo for pip
 RUN echo -e "[epel]\nname=epel\nenabled=1\nbaseurl=https://dl.fedoraproject.org/pub/epel/7/x86_64/\ngpgcheck=0" > /etc/yum.repos.d/epel.repo
 
 # lets install pip, and gcc for the native extensions
 # and remove all after use
-RUN dnf install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc && \
-    python setup.py install && \
-    pip install -r ./requirements.txt && \
-    dnf remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
+RUN dnf install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc
```
Comment (vpavlin) [inline Dockerfile.fedora]: Same as above - missing `dnf clean all`

File: Dockerfile.fedora [hunk 5]
```diff
@@ -1,28 +1,26 @@
...
+RUN python setup.py install
+RUN dnf remove -y gcc cpp glibc-devel glibc-headers kernel-headers libmpc mpfr python-pip && \
```
Comment (goern) [inline Dockerfile.fedora]: and pls consolidate all RUNs in one RUN line, so we will create one layer and have it cleaned up in the end

File: Dockerfile (hunk 2 above)
Comment (goern) [inline Dockerfile]: caching seems to be irrelevant for a base image, I would go for small size rather than the time needed to build to image.

**Comment by charliedrage** [comment]
Will be re-doing a cleaner pull request**
